### PR TITLE
Use obs assets

### PR DESCRIPTION
**Pull request by ChrisBr**

### CODE REVIEW

**Comment by hennevogel** [comment]
Deploy preview for *obs-patterns* ready!

Built with commit 6da817a940a88e4868f0f6d36b42523fe6aa7a5d

https://deploy-preview-33--obs-patterns.netlify.com

**Comment by dmarcoux** [comment]
This depends on https://github.com/openSUSE/open-build-service/pull/6128

File: _includes/head.html [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css" integrity="sha384-WskhaSGFgHYWDcbwN70/dfYBj47jz9qbsMId/iRN3ewGhXQFZCSftd1LZCfmhktB" crossorigin="anonymous">
-  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.1/css/all.css" integrity="sha384-O8whS3fhG2OnA5Kas0Y9l3cfpmYjapjI0E4theH4iuMD+pLhbf6JI0jIMfYcK3yZ" crossorigin="anonymous">
   <title>OBS patterns</title>
   <!-- Custom styles for this template -->
   <link href="/assets/main.css" rel="stylesheet">
   <link href="/assets/syntax.css" rel="stylesheet">
 
-  <script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
+  <link rel="stylesheet" media="screen" href="https://build.opensuse.org/assets/webui2/webui2.css" />
```
Comment (coolo) [inline _includes/head.html]: you better invalidate caching here - or it will be very painful to watch as the assets evolve.

**Review by bgeuken — APPROVED** [review]
LGTM, :thumbsup:

**Comment by ChrisBr** [comment]
Good idea with the timestamp @dmarcoux

**Comment by dmarcoux** [comment]
Oh, the icons are broken... because we removed font-awesome.

**Pull request merged**